overlap-scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -77,19 +77,33 @@ then kubectl ws "${kubectl_flags[@]}" "$wmw_name"
 else kubectl ws "${kubectl_flags[@]}" create "$wmw_name" --enter
 fi
 
-if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-espw &> /dev/null; then
-kubectl "${kubectl_flags[@]}" apply -f - <<EOF
-apiVersion: apis.kcp.io/v1alpha1
-kind: APIBinding
-metadata:
-  name: bind-espw
-spec:
-  reference:
-    export:
-      path: root:espw
-      name: edge.kubestellar.io
-EOF
-fi
+function run_kb_konnector_for_wmw() {
+    if kubectl get ns kube-system &> /dev/null; then
+        echo "namespace kube-system already exists, skipping"
+    else
+        echo "creating namespace kube-system"
+        kubectl create ns kube-system
+    fi
+    echo "starting kube-bind konnector for wmw in background"
+    konnector &
+    if ! pgrep konnector &> /dev/null; then
+        echo "kube-bind konnector not started"
+        exit 1
+    fi
+}
+
+function bind_resource_for_wmw() {
+    RESOURCE="$1"
+    echo "binding $RESOURCE for wmw"
+    if ! kubectl bind http://127.0.0.1:8080/export --skip-konnector --unattended --resource "$RESOURCE" ; then
+        echo "unable to bind $RESOURCE for wmw"
+        exit 1
+    fi
+}
+
+run_kb_konnector_for_wmw
+bind_resource_for_wmw "edgeplacements"
+bind_resource_for_wmw "customizers"
```

Comment:
What about the rest of the edge API?

Reply:
We now added sps for workload description spaces. The rest 4 (synctargets, locations, syncerconfigs and edgesyncconfigs) are not needed by WDSes.